compliance/aws/untagged_resources/aws_untagged_resources.pt

```diff
@@ -0,0 +1,273 @@
+        end
+      end
+      field "region", val(iter_item,"region")
+    end
+  end
+end
+
+datasource "ds_filter_resources" do
+  run_script $js_filter_resources, $ds_get_all_resources, $param_tags_to_check
+end
+
+###############################################################################
+# Scripts
+###############################################################################
+
+script "js_filter_resources", type: "javascript" do
+  parameters "ds_get_all_resources", "param_tags_to_check"
+  result "result"
+  code <<-EOS
+  var result=[];
+  for(var i=0;i<ds_get_all_resources.length;i++){
+    var missing_tags="";
+    var all_tags="";
+    var instance=ds_get_all_resources[i];
+    for(var j=0;j<param_tags_to_check.length;j++){
+      var param_lowerCase=param_tags_to_check[j].toLowerCase();
+      var notMatched=true;
+      if(param_lowerCase.indexOf("=")==-1)
+      {
+        for(var k=0;k<instance.tags.length;k++){
+          var current_tag=instance.tags[k]; 
+          if(current_tag["tagKey"].toLowerCase()==param_lowerCase || current_tag["tagValue"].toLowerCase()==param_lowerCase){
+            notMatched=false;
+            break;
+          }
+        }
+      }
+      else {
+        var arr=param_lowerCase.split("=");
+        for(var k=0;k<instance.tags.length;k++){
+          var current_tag=instance.tags[k];
+          if(current_tag["tagKey"].toLowerCase()==arr[0] && current_tag["tagValue"].toLowerCase()==arr[1]){
+            notMatched=false;
+            break;
+          }
+        }
+      }
+      if(notMatched){
+        missing_tags=missing_tags+", "+param_tags_to_check[j];
+      }
+    }
+    for(var j=0;j<instance.tags.length;j++){
+      var current_tag=instance.tags[j];
+      all_tags=all_tags+", "+current_tag["tagKey"]+"="+current_tag["tagValue"];
+    }
+    if(all_tags.length==0){
+      all_tags="< No Value >";
+    }
+    else{
+      all_tags=all_tags.slice(2);
+    }
+    var id="";
+    var type="";
+    var arn_last=instance.arn.split(":");
+    if(arn_last[arn_last.length-1].indexOf("/")!=-1){
+      var arr=arn_last[arn_last.length-1].split("/")
+      type=arr[0];
+      id=arr[1];
+    }
+    else{
+      type="< Not Available >";
+      id=arn_last[arn_last.length-1];
+    }
+    if(missing_tags.length!=0){
+      result.push({
+        "arn":instance.arn,
+        "id": id,
+        "resource_type": type,
+        "missing_tags":missing_tags.slice(2),
+        "existing_tags":all_tags,
+        "region":instance.region
+      })
+    }
+  }
+  result=_.sortBy(result, 'id');
+  result=_.sortBy(result, 'region');
+  EOS
+end
+
+###############################################################################
+# Escalations
+###############################################################################
+
+escalation "esc_email" do
+  automatic true
+  label "Sends Email"
+  description "Sends Incident Email"
+  email $param_email
+end
+
+escalation "esc_add_tags" do
+  automatic false
```

Comment:
Missing the label and description

Reply:
added the label and the description.

Reply:
@KVivek03   Can you please check the label?  The escalation/action label is probably better stated as Add Tags.

Reply:
modified the policy